model/pom.xml

```diff
@@ -96,19 +96,29 @@
 
   <build>
     <plugins>
-      <!-- Creates (or re-creates) a keywhizdb_test database. -->
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
-        <artifactId>sql-maven-plugin</artifactId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>3.3.0</version>
         <executions>
           <execution>
-            <phase>initialize</phase>
+            <phase>generate-sources</phase>
             <goals>
-              <goal>execute</goal>
+              <goal>add-source</goal>
             </goals>
+            <configuration>
+              <sources>
+                <source>src-generated/main/java</source>
+              </sources>
+            </configuration>
           </execution>
         </executions>
+      </plugin>
```

Comment:
GitHub really whiffed on the diff here. Maven doesn't have first-class support for multiple source directories, so since we'd like to keep the hand-written and generated code in separate folders we're using this plugin (which appears to be the standard approach to solving this problem).

I picked src-generated because it will appear alphabetically next to the usual src folder but also contains the word "generated".

Reply:
seems reasonable 👍